ch5-catdir/controller.cpp

```diff
@@ -50,7 +51,17 @@ void Controller::process(const std::string_view& path) try
     auto dir = Directory(path);
     while (auto ent = dir.next())
     {
-        m_out << std::setw(8) << ent->m_inode << ' ' << ent->m_name << '\n';
+        if (ent->m_extra)
+        {
+            m_out << std::setw(8) << ent->m_extra->m_inode << ' '
+                  << fmtMode(ent->m_extra->m_mode) << ' '
+                  << std::setw(3) << ent->m_extra->m_linkCount << ' ';
+        }
+        else
+        {
+            m_out << "     <?>        <?> <?> ";
```

Comment:
Consider using the same `setw()` calls here too for better readability and maintainability.

Reply:
I did it.